index.html

```diff
@@ -0,0 +1,837 @@
+            <p class="link">Sunday Review</p>
+            <p class="link">Video: Opinion</p>
+  
+          </nav>
+  
+          <nav class="column">
+  
+            <h3 class="titlelink link">ARTS</h3>
+            <p class="link">Today's Arts</p>
+            <p class="link">Art & Design</p>
+            <p class="link">Books</p>
+            <p class="link">Dance</p>
+            <p class="link">Movies</p>
+            <p class="link">Music</p>
+            <p class="link">Pop Culture</p>
+            <p class="link">Television</p>
+            <p class="link">Theater</p>
+            <p class="link">Video: Arts</p>
+  
+          </nav>
+  
+          <nav class="column">
+  
+            <h3 class="titlelink link">LIVING</h3>
+            <p class="link">At Home</p>
+            <p class="link">Automobiles</p>
+            <p class="link">Games</p>
+            <p class="link">Education</p>
+            <p class="link">Food</p>
+            <p class="link">Health</p>
+            <p class="link">Jobs</p>
+            <p class="link">Love</p>
+            <p class="link">Magazine</p>
+            <p class="link">Parenting</p>
+            <p class="link">Real Estate</p>
+            <p class="link">Recipes</p>
+            <p class="link">Style</p>
+            <p class="link">T Magazine</p>
+            <p class="link">Travel</p>
+  
+          </nav>
+  
+          <nav class="column">
+  
+            <h3 class="titlelink link">MORE</h3>
+            <p class="link">Reader Center</p>
+            <p class="link">Wirecutter</p>
+            <p class="link">Live Events</p>
+            <p class="link">The Learning Network</p>
+            <p class="link">Tools & Services</p>
+            <p class="link">Multimedia</p>
+            <p class="link">Photopraphy</p>
+            <p class="link">Video</p>
+            <p class="link">Newsletters</p>
+            <p class="link">TimesMachine</p>
+            <p class="link">NYT Store</p>
+            <p class="link">Times Journeys</p>
+            <p class="link">Manage My Account</p>
+  
+          </nav>
+  
+          <nav class="column bordercol">
+  
+            <h3 class="titlelink link">SUBSCRIBE</h3>
+            <p class="link2"><img src="assets/footer1.png" alt="delivery img">Home Delivery</p>
+            <p class="link2"><img src="assets/footer2.png" alt="digital imag">Digital Subscriptions</p>
+            <p class="link2"><img src="assets/footer3.png" alt="games img">Games</p>
+            <p class="link2"><img src="assets/footer4.png" alt="cook img">Cooking</p>
+            <br>
+            <p class="link">Email Newsletters</p>
+            <p class="link">Corporate Subscriptions</p>
+            <p class="link">Education Rate</p>
+
+            <hr>
+            
+            <p class="link">Mobile Applications</p>
+            <p class="link">Replica Edition</p>
+            <p class="link">Español</p>
+            <p class="link">中文网</p>
+          
+          </nav>
+  
```

Comment:
All the `p` tags, with a class of link, should be links as it is in the original